<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleInfoWgt</class>
 <widget class="QWidget" name="VehicleInfoWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>1057</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="TGFrame" name="rightTitleFrame1" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>791</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>771</width>
         <height>369</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="map_widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="Dashboard" name="vehicleWgt" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(78, 154, 6);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Dashboard" name="vehicleWgt_2" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(138, 226, 52);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="Dashboard" name="vehicleWgt_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 185, 110);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Dashboard" name="vehicleWgt_4" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(117, 80, 123);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="TGFrame" name="rightTitleFrame2" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>850</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="sysInfoWgt" native="true">
      <widget class="QTextBrowser" name="sysInfo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background:transparent;
border:none;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TGFrame</class>
   <extends>QWidget</extends>
   <header>Component/TGFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Dashboard</class>
   <extends>QWidget</extends>
   <header>Component/Dashboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
